userguide/dailyuse/libertine.rst

```diff
@@ -72,7 +72,7 @@ From the apps list in the *Manage Libertine Containers* menu you can tap the + s
 
 Enter the package name if you know it.
 Otherwise you can search the archives for a package.
-You can make this process easier by making the *Lubuntu Software Center* your first package install.
+You can make this process easier by making a software center like gnome-software your first package install.
```

Comment:
I'm wondering ... did you try to install gnome-software? is it possible? does it run? does it run well enough to justify the claim that this makes the "process easier"?

It's been a long long time that I looked at this, but I think gnome-software is a rather .... whatstheword .... advanced ... sophisticated ... modern application. I'm wondering how well it actually runs in libertine. did you test it? can you comment?

Reply:
I did test it, it worked fine though I would recommend attaching a mouse for ease of use (I can't get the scaling to work yet)

Reply:
Alternatively feel free to suggest another one, the one to change it *to* isn't really the point of this PR

Reply:
> I did test it, (fp4 focal ota-4) it works fine though I would recommend attaching a mouse for ease of use (I can't get the scaling to work yet)

great! thanks